lib/drb/ssl.rb

```diff
@@ -185,7 +185,7 @@ def setup_certificate
         }
 
         cert = OpenSSL::X509::Certificate.new
-        cert.version = 3
+        cert.version = 2
```

Comment:
This is a great catch @jeremyevans but I'm afraid someone will come along and break this later.
A tiny comment like `# This means v3` would be super valuable.

Reply:
You would assume someone changing something like this would use `git blame` to look at the history, but I agree that a comment would make things easier.  As this has already been merged, can you submit a pull request to add the comment?

Reply:
Happy to do it. Cheers.